webpack/components/Errata/index.js

```diff
@@ -5,8 +5,62 @@ import {
   BugIcon,
   SecurityIcon,
   EnhancementIcon,
+  SquareIcon,
 } from '@patternfly/react-icons';
 import PropTypes from 'prop-types';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+
+export const ErrataMapper = ({ data, id }) => data.map(({ x: type, y: count }) => <ErrataSummary count={count} type={type} key={`${count} ${type}`} id={id} />);
+
+export const ErrataSummary = ({ type, count, id }) => {
+  let ErrataIcon;
+  let label;
+  let name;
+  let url;
+  let color;
+  switch (type) {
+    case 'security':
+      label = __('Security');
+      ErrataIcon = SecurityIcon;
+      name = 'security advisories';
+      color = '#0066cc';
+      url = <a href={urlBuilder(`content_hosts/${id}/errata`, '')}> {count} {name} </a>;
```

Comment:
I'd say let's link to the new page for now. Since we're staying in React land, we can
```js
import { Link } from 'react-router-dom';
```
and then
```suggestion
      url = <Link to={urlBuilder(`Content/Errata`, '')}> {count} {name} </Link>;
```
Apparently the host ID isn't needed in the url with `<Link>` because it only changes the part of the url after the `#`. Added bonus!